scania-digital-design-system/tegel
# fix(datetime): corrected custom event

**Describe pull-request**  
We have been using tdsChange for both onInput and onChange. For those users that listen to tdsChange, it was making a lot of emits at the same time as onChange and onInput happen at the same time. 

**Solving issue**  
Fixes: [CDEP-3121](https://tegel.atlassian.net/browse/CDEP-3121)
Possible improvement for: https://github.com/scania-digital-design-system/tegel/issues/510

**How to test**  
1. Go to preview link and click around the component.
2. Confirm you get tdsFocus, tdsInput, tdsChange and tdsBlur events on component.

**Checklist before submission**



[CDEP-3121]: https://tegel.atlassian.net/browse/CDEP-3121?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
